pkg/ring/model.go

```diff
@@ -434,12 +450,12 @@ func (d *Desc) getTokensByZone() map[string][]TokenDesc {
 type CompareResult int
 
 const (
-	Equal                   CompareResult = iota // Both rings contain same instances, tokens, states and timestamps.
-	EqualInstancesAndTokens                      // Both rings contain the same instances and tokens, but states and timestamps differ.
-	Different                                    // Rings have different set of instances, or their information don't match.
+	Equal                          CompareResult = iota // Both rings contain same exact instances.
+	EqualUnlessStatesAndTimestamps                      // Both rings contain the same instances with the same data except states and timestamps (may differ).
```

Comment:
`EqualButStatesAndTimestamps` or `EqualExceptStatesAndTimestamps`

Reply:
Ok. Picked `EqualButStatesAndTimestamps` just because it's slower.